### ISSUE TITLE
resnet101_c.pth.tar

### ISSUE THREAD

**Issue by bad-meets-joke** [wenjiaXu/APN-ZSL#11]
Hi， 

In `mode/model_proto.py, line 76-78`,  the file for loading  pretrained backbone weights is **resnet101_c.pth.tar**

My question is, what is the difference between **resnet101_c.pth.tar** and default **resnet101-5d3b4d8f.pth** given by pytorch?

**Comment by wenjiaXu** [comment]
Hi, for CUB dataset we can start from the ResNet model pre-trained on Seen classes with cross-entropy loss, i.e. resnet101_c.pth.tar, which would fasten the model training. resnet101-5d3b4d8f.pth is the original ResNet model pre-trained on ImageNet.

**Comment by in-my-heart** [comment]
> Hi, for CUB dataset we can start from the ResNet model pre-trained on Seen classes with cross-entropy loss, i.e. resnet101_c.pth.tar, which would fasten the model training. resnet101-5d3b4d8f.pth is the original ResNet model pre-trained on ImageNet.

Do the results of the paper report use a backbone that is fine-tuned on the Seen classes?

**Comment by wenjiaXu** [comment]
> > Hi, for CUB dataset we can start from the ResNet model pre-trained on Seen classes with cross-entropy loss, i.e. resnet101_c.pth.tar, which would fasten the model training. resnet101-5d3b4d8f.pth is the original ResNet model pre-trained on ImageNet.
> 
> Do the results of the paper report use a backbone that is fine-tuned on the Seen classes?

Hi,
Yes, in this paper we use the seen class train set images to train the backbone.

We also compare our features with the features finetuned with seen class train set images [1] (see the last four lines in Table 2), and prove that we yields much better results than [1].

[1] https://www.mpi-inf.mpg.de/departments/computer-vision-and-machine-learning/research/zero-shot-learning/f-vaegan-d2-a-feature-generating-framework-for-any-shot-learning

**Comment by in-my-heart** [comment]
> > > Hi, for CUB dataset we can start from the ResNet model pre-trained on Seen classes with cross-entropy loss, i.e. resnet101_c.pth.tar, which would fasten the model training. resnet101-5d3b4d8f.pth is the original ResNet model pre-trained on ImageNet.
> > 
> > 
> > Do the results of the paper report use a backbone that is fine-tuned on the Seen classes?
> 
> Hi, Yes, in this paper we use the seen class train set images to train the backbone.
> 
> We also compare our features with the features finetuned with seen class train set images [1] (see the last four lines in Table 2), and prove that we yields much better results than [1].
> 
> [1] https://www.mpi-inf.mpg.de/departments/computer-vision-and-machine-learning/research/zero-shot-learning/f-vaegan-d2-a-feature-generating-framework-for-any-shot-learning

When I read this paper Goal-Oriented Gaze Estimation for Zero-Shot Learning(CVPR2021), I think there are many similarities with your article, his inspiration may come from your paper.
